swan-lake/integration-tools/health-tool.md

```diff
@@ -36,132 +35,73 @@ The parameters that are available with the tool are listed below.
 | `-v, --version`  | Print the version information of the Health tool.                                                                                                                                                                                                                                                                                                                               | optional           |
 | `-h, --help`     | Print the usage details of the Health tool.                                                                                                                                                                                                                                                                                                                                     | optional           |
 
-## Example
+## Install the tool
 
-Follow the steps below to try out an example of using the Health tool.
-
-1. Pull the Health tool from Ballerina Central.
+Pull the Health tool from Ballerina Central.
 
     ```
     $ bal tool pull health
     health:1.0.0 pulled successfully.
     health:1.0.0 successfully set as the active version.
     ```
 
-2. Download the **JSON Definitions ZIP archive** from the <a href="http://hl7.org/fhir/us/carin-bb/STU2/downloads.html" target="_blank">Carin BB implementation guide</a> and extract it to your current directory.
+## Create the package
+
+Follow the steps below to create the package.
 
-3. Run the tool with the [required arguments](#available-parameters).
+1. Download the **JSON Definitions ZIP archive** from the [Carin BB implementation guide](http://hl7.org/fhir/us/carin-bb/STU2/downloads.html) and extract it to your current directory.
+
+2. Run the tool with the [required arguments](#available-parameters) to generate the package.
 
     >**Note:** Provide values for mandatory arguments. Make sure to give the directory path of the downloaded FHIR definitions as the last argument.
 
     ```
-    $ bal health fhir -m package --org-name healthcare --package-name carinbb.lib -o gen ./definitions.json
+    $ bal health fhir -m package --org-name healthcare_samples --package-name carinbb_package -o gen ./definitions.json
     Ballerina FHIR package generation completed successfully.
     ```
 
-4. Build the generated package and push it to a repository.
+3. Build the generated package.
 
     >**Info:** You can push either to the local repository or the remote repository in Ballerina Central.
```

Comment:
This info section is not required here since it is not related to the build step.
In the `push` step, same information is mentioned as a tip. (L79)

Reply:
Fixed.